src/components/common/Input/Input.tsx

```diff
@@ -29,17 +31,28 @@ const Input: React.FC<IInput> = ({
   register,
   errors,
   name,
-  placeholder,
   accept,
   id,
   fileName,
-  value,
+  onChange,
   ...props
 }) => {
+  const [inputFileName, setInputFileName] = useState('Selected photo');
+  const fileOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    e.target?.files && setInputFileName(e.target.files[0].name);
+  };
   return (
-    <InputGroup fileName={fileName} primary={primary} type={type}>
-      <label htmlFor={id}>{fileName ? fileName : label}</label>
-      <input id={id} name={name} type={type} value={value} {...register} {...props} />
+    <InputGroup inputFileName={inputFileName} fileName={fileName} primary={primary} type={type}>
+      <label htmlFor={id}>{type === 'file' ? inputFileName : label}</label>
```

Comment:
what's the point of overriding styles in parent component instead of creating just a separate styled component for label?
```
const Label = styled.label``;
```

Reply:
same about input

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/27/commits/42e7f80a02b408b3240bbe5ae57d6f746a5ac154#diff-95a9cb97fb3fbf2b1721aa6937c75e6deb40e9e791fd2ab6d688dd0fae80e111

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/27/commits/91ffd5bf7229772ab21b7b0c7afae094de09037b#diff-e2eec15dd4a882dc8cd779ef821494357bd0dface0e4d8e40fd8d7c4f8bbe31a